dts/arm/nxp/nxp_rt6xx_common.dtsi

```diff
@@ -138,6 +138,22 @@
 		label = "FLEXCOMM_7";
 		status = "disabled";
 	};
+
+	hs_lspi: spi@126000 {
+		compatible = "nxp,lpc-spi";
+		/* Enabling cs-gpios below will allow using GPIO CS,
+		 rather than Flexcomm SS */
+		/* cs-gpios = <&gpio1 14 GPIO_ACTIVE_LOW>,
```

Comment:
see comment in pinmux.c

Reply:
Thank you, this is good to know. In this case, the configuration is for FlexComm5 and not HS_LSPI, hence we need not apply this.

Reply:
@mmahadevan108 now i see there are different spi ports.